packages/libvips/build.sh

```diff
@@ -18,4 +19,15 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 termux_step_pre_configure() {
 	termux_setup_gir
 	termux_setup_glib_cross_pkg_config_wrapper
+
+	export TERMUX_MESON_ENABLE_SOVERSION=1
+}
+
+termux_step_post_massage() {
+	# Do not forget to bump revision of reverse dependencies and rebuild them
+	# after SOVERSION is changed.
+	local _GUARD_FILE="lib/libvips.so.42"
+	if [ ! -e "${_GUARD_FILE}" ]; then
+		termux_error_exit "Error: file ${_GUARD_FILE} not found."
+	fi
```

Comment:
@TomJo2000's suggestion, but with errors printed to `stderr` and without the bash-specific `[[ ]]` brackets.

```suggestion
	local _SOVERSION=42
	if [ ! -e "lib/libvips.so.${_SOVERSION}" ]; then
		echo "ERROR: Expected: lib/libvips.so.${_SOVERSION}" 1>&2
 		echo "ERROR: Found   : $(find lib/libvips* -regex '.*so\.[0-9]+')" 1>&2
 		termux_error_exit "Not proceeding with update." 
	fi
```

Reply:
Our build script for ran under Bash, so `[[  ]]` is preferred usually but it's not a blocker.
Printing those errors to `stderr` is a good catch, thank you.